### DISCUSSION TITLE
Running AppImages

### DISCUSSION THREAD

**Discussion by doobes** [elementary/triage#609] [Q&A]
So,

I installed v8.0.  

Several regressions from v7.1 have made themselves apparent, the most egregious at this point are the issues running AppImages.

I've done some digging, and allegedly it's an Ubuntu Fuse 2 vs Fuse 3 thing. The solution is to install Fuse 2.  Trouble is, it's already installed. AppImages still won't run.

 As It stands now, I have to run:

```
sudo sysctl -w kernel.apparmor_restrict_unprivileged_unconfined=0
sudo sysctl -w kernel.apparmor_restrict_unprivileged_userns=0
```

Every time I reboot, or create new configs for each AppImage....

So, what is the solution?

And it's not "don't run AppImages".....

**Comment by danirabbit** [comment]
This is an upstream issue in AppImage that occurs on basically all modern distributions. This needs to be fixed in AppImage.

also AppImage is not a supported application format, sorry. The only supported format for sideloading apps is Flatpak

**Comment by doobes** [comment]
Well,

That's all fine and good, but unfortunately there are AppImages that people use and rely on.

**Comment by teamcons** [comment]
Yes. And Ubuntu itself doesnt ship it by default either.


> sudo add-apt-repository universe
> sudo apt install libfuse2t64

if not try

> sudo apt install libfuse3-3

or

>  sudo apt install libfuse2


This is not a regression. This is like people expecting Java or Wine preinstalled. This is you expecting your case to be covered.
No distribution can do everything for everyone. 
If you think it should be default there for everyone, you need to involve yourself in design decisions, not make angry remarks like it is self-evident.

I too run appimages. If this distribution had it out of the box that would be "nice to have" not "expected". The solution was three seconds of googling and documented by the Appimage devs themselves.

Please next time ask "please"

**Comment by doobes** [comment]
Well,

I stood up a Ubuntu 24.02 LTS VM, and my applications run under that. Read into that what you may.....

The final answer is here: https://askubuntu.com/questions/1511654/how-to-make-balena-etcher-and-other-appimages-work-in-ubuntu-24-04

**Reply by teamcons** [reply]
Yes, this is about electron apps.
The fix is for electron apps.
I provided you with a fix for the appimage format. Because you asked for appimage.
I didnt know your apps were electron ones.

Next time i will not be interested in helping you with that attitude. Not worth the time.
